libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Media/AudioSender.cs

```diff
@@ -197,8 +197,9 @@ internal override async Task AttachTrackAsync()
             //< FIXME - Multi-track override!!!
             Transceiver.PeerConnection.PreferredAudioCodec = PreferredAudioCodec;
 
-            // Ensure the local sender track exists
-            if (Track == null)
+            // Ensure the local sender track exists, but do not create
+            // if the component is not active.
+            if ((Track == null) && isActiveAndEnabled)
```

Comment:
nit: if the component is enabled after `AttachTrackAsync` I suppose we should create the track in `OnEnable`. Or at least we should ensure that what happens if one does that is deterministic and documented.